### ISSUE TITLE
Reaction for Spell not Showing When Out of Pact Magic Spell Slots

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#766]
In GitLab by @JamesBrandts on Mar 16, 2022, 10:09

I have in my game a multiclass actor with the spell Shield, despite it being in the 1st Level spells list, not in the Pact Magic Spells list, when the actor receives an attack, that would hit, the reaction Dialog does not show, if the actor is out of Pact Magic spell slots, even if it has 1st level spell slots.

[fvtt-midi-qol-settings.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/29838b98486702873118bec6cc97ffb799768b2e69989c8a9ce8ffae92061a72/fvtt-midi-qol-settings.json)

**Comment by tposney** [comment]
I tried to reproduce this with an multi classed actor but could not - it seemed to behave correctly.
Can you export the actor and attach it to the issue. That might give me a better idea what is going on.

**Comment by tposney** [comment]
In GitLab by @JamesBrandts on Mar 18, 2022, 07:25

I created a new world and activated only Midi-QoL and DAE, to be sure it wasn't other module interfering, and got the same result. I observed this happening in more than a world I have created, may be there's something in my actors or spells.

This is the actor on the current game:
[fvtt-Actor-baal-zebub.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/6fe2e2acfaa5284fa1171c8358998c103c9c3f058063ef700cb27389aac6a951/fvtt-Actor-baal-zebub.json)

This is the actor I observed this happening for the fist time:
[fvtt-Actor-hexadin.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/b45c2c99f5895f5ba36e93ebd051f643afccf7ef875b9bd08cab19e17c4bd7ea/fvtt-Actor-hexadin.json)

This 2 I created, just for test, I a newly created world, with only Midi-QoL and DAE activated, tried with shield as Sorcerer Spell, as Pact Magic, same results, no pact magic = no reaction Dialog:
[fvtt-Actor-sorcerer-1st.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/04f481b488a346e70e4c09b9455eaf54e97209a78c7e9f492eddf27e3f31edb9/fvtt-Actor-sorcerer-1st.json)
[fvtt-Actor-warlock-1st.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/d0feecbdba0fbf49bc2d37636f08a72969e36ae484e7ae21796dc6dfcd564f17/fvtt-Actor-warlock-1st.json)

Thanks for the your attention, I don't know what I would do without this amazing module.
